www/src/pages/en/usage/next-auth.md

```diff
@@ -4,7 +4,7 @@ description: Usage of NextAuth
 layout: ../../../layouts/docs.astro
 ---
 
-For when you want an authentication system in your Next.js application, NextAuth.js is a perfect solution to bring in the complexity of security without the hassle of having to build it yourself. It comes with an extensive list of providers to quickly add OAuth authentication, as well as a database adapter system to allow you to use your own database of choice.
+When you want an authentication system in your NextJS application, NextAuth.js is an excellent solution to bring in the complexity of security without the hassle of having to build it yourself. It comes with an extensive list of providers to quickly add OAuth authentication, provides adapters for many databases and ORMs, and lets you integrate an existing authentication system by using the `CredentialsProvider`.
```

Comment:
it's `Next.js`?

Reply:
so we want to advocate the use of the `CredentialsProvider`?

Reply:
> also, do we want to advocate the use of the `CredentialsProvider`?

I think we should not

Reply:
Re: Next.js - Vercel is a bit inconsistent on the branding, but this is seems to be the most common way of writing it.
<img width="637" alt="image" src="https://user-images.githubusercontent.com/8353666/199087948-81779550-57fd-4ee6-b649-bc269b26827f.png">

Reply:
also removed the credentialsprovider reference in both `why.md` and `next-auth.md`